src/geometry/manifold/tangent_bundle.lean

```diff
@@ -32,16 +32,16 @@ not require that this changes in fiber are linear, but only diffeomorphisms.
 
 ## Main definitions
 
-* `basic_smooth_bundle_core I M F`: assuming that `M` is a smooth manifold over the model with
+* `smooth_vector_bundle_core I M F`: assuming that `M` is a smooth manifold over the model with
```

Comment:
There is a subtlety here. Normally, `smooth_vector_bundle_core` should correspond to some set of trivializations of a bundle, for which the changes of coordinates should be smooth. The current definition we have here is less general: it requires that the trivialization sets correspond exactly to the chart domains for the atlas of the manifold (but it makes for better defeq properties). This is why I had used the word `basic`, to emphasize that it was not the general notion. But I agree that my name is not good, as it does not highlight that we are building a vector bundle structure. What would you think of `basic_smooth_vector_bundle_core`?

Reply:
Sure the name looks good to me. Just to understand well though, here the construction you implemented could not be made with what would correctly be called `smooth_vector_bundle_core`, or it can but it would just create additional problems that would need to be solved?

Reply:
It could be done with the general version of `smooth_vector_bundle_core`, but it would be more painful and slightly less convenient in the end (for instance, the chart at `(x, v)` would have no reason to be the derivative of the chart at `x`).